### PR TITLE
Publish sensor-board color reading 

### PR DESCRIPTION
**Pull request by nuft**

### CODE REVIEW

File: uavcan_data_types/cvra/sensor/20141.ColorTCS3472.uavcan [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
```
Comment (SyrianSpock) [inline uavcan_data_types/cvra/sensor/20141.ColorTCS3472.uavcan]: why not call it simply `20141.Color.uavcan`?

Comment (nuft) [reply uavcan_data_types/cvra/sensor/20141.ColorTCS3472.uavcan]: Right, this message is quite generic. Originally, I wanted to add sensor specific configuration parameters, which became too complicated.
How about `20141.ColorRGBC.uavcan` or `20141.ColorRGBC16.uavcan`?

Comment (SyrianSpock) [reply uavcan_data_types/cvra/sensor/20141.ColorTCS3472.uavcan]: `20141.ColorRGBC.uavcan` sounds good to me

Comment (nuft) [reply uavcan_data_types/cvra/sensor/20141.ColorTCS3472.uavcan]: done

**Pull request merged**